temoa/extensions/monte_carlo/mc_sequencer.py

```
@@ -39,16 +39,34 @@ class MCSequencer:
                     all_options = tomllib.load(f)
+            else:
+                with resources.as_file(solver_options_path) as path:
+                    with open(path, 'rb') as f:
+                        all_options = tomllib.load(f)
             s_options = all_options.get(self.config.solver_name, {})
             logger.info('Using solver options: %s', s_options)
 
         except FileNotFoundError:
-            logger.warning('Unable to find solver options toml file.  Using default options.')
+            if options_file_path:
+                logger.error('Unable to find custom solver options file at: %s', options_file_path)
```

Ruff (0.14.10)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)